examples/influxDB2-nifi/deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
if this file is no different that the default deploy.sh, use the defautl one. 
Otherwise it needs some comments on what has been changed.

Reply:
the deploy.sh script had an error. I just saw that an update has been done by Zakaria.